.storybook/config.js

```diff
@@ -1,11 +1,10 @@
 import React from 'react';
-import { configure, addDecorator } from '@storybook/react';
+import { configure, addDecorator, addParameters } from '@storybook/react';
 import { withKnobs } from '@storybook/addon-knobs';
-import { withNotes } from '@storybook/addon-notes';
```

Comment:
I think we're using this addon to display the Design Notes, does another plugin replace it? Or are you planning on replacing it with anything?

Reply:
Hi Dave, I'm not removing this addon. It doesn't need to be imported here anymore.